app/controllers/insights_cloud/ui_requests_controller.rb

```diff
@@ -75,6 +75,18 @@ def assign_header(res, cloud_res, header, transform)
       res.headers[new_header] = header_content
     end
 
+    def translate_insights_host
+      Rails.logger.info "translate_insights_host"
+      facet = InsightsFacet.find_by(uuid: params[:uuid])
+      if facet.present?
+        Rails.logger.info "Found InsightsFacet #{params[:uuid]}"
```

Comment:
```suggestion
        Rails.logger.debug "Found InsightsFacet #{params[:uuid]}"
```
I don't think we need to display this message if we don't debug the system.